templates/set-absolute-and-idle-timeout-based-on-organization-connection-or-application-metadata-POST_LOGIN/manifest.yaml

```diff
@@ -0,0 +1,28 @@
+id: "454ef310-4779-41bb-bed4-c916ab9eaaed"
+name: "Set Absolute and Idle Timeout based on Organization, Connection or Application Metadata"
+triggers:
+	- "POST_LOGIN"
+useCases:
+	- 
+public: true
+description: "Configure a custom absolute or idle timeout based on specific metadata that can be create either on organization, connection or application"
+version: "1.0.0"
+runtime: "node18"
+sourceUrl: "https://github.com/auth0/opensource-marketplace/tree/main/templates/set-absolute-and-idle-timeout-based-on-organization-connection-or-application-metadata-POST_LOGIN/manifest.yaml"
+notes: ""
```

Comment:
Given that some related features are in EA, would it worth having notes to that effect until the features hit GA?

Reply:
We've announced GA yesterday, we are waiting a merge to fix the tooltip and documentation but the features are fully available